tests/fixtures.py

```diff
@@ -395,14 +402,22 @@ def get_data_path_by_name(name) -> Path:
             / "v2.2"
             / "d46900384c78863420fb3e297d0a2f743cd2b6b3f7f82bf64059a168e07aceb7.json.gz"
         )
-    elif name.startswith("93b2d1"):
+    elif name.startswith("93b2d1-drakvuf"):
         return (
             CD
             / "data"
             / "dynamic"
             / "drakvuf"
             / "93b2d1840566f45fab674ebc79a9d19c88993bcb645e0357f3cb584d16e7c795.log.gz"
         )
+    elif name.startswith("93b2d1-vmray"):
+        return (
+            CD
+            / "data"
+            / "dynamic"
+            / "vmray"
+            / "93b2d1840566f45fab674ebc79a9d19c88993bcb645e0357f3cb584d16e7c795_archive.zip"
```

Comment:
pending, I have an archive ready (created via `scripts/minimize_vmray_results.py` part of this PR)